src/ui/screens/DashboardScreen.tsx

```diff
@@ -54,6 +55,7 @@ export function DashboardScreen({navigation, route}: Props) {
   return (
     <Layout style={styles.container}>
       <MainAppBar navigation={navigation} route={route} options={appBarOptions} />
+      <NetworkConnectionError />
```

Comment:
Also here, is it still needed?

Reply:
Yes, we need it.

Reply:
Why do we need it explicitly in the dashboard screen?

Reply:
Because the screen stack is different,adding there would be present on all the dashboard screen.

Reply:
I am not sure what you mean. We want this banner in all the screens.

Reply:
Yes, but since there are two significant screens, one is to render the dashboard, and another one to render other pages. Adding here gives the banner to the dashboard.